tests/langchain/test_langchain_model_export.py

```diff
@@ -229,7 +230,7 @@ def test_langchain_model_predict():
             logged_model = mlflow.langchain.log_model(model, "langchain_model")
         loaded_model = mlflow.pyfunc.load_model(logged_model.model_uri)
         result = loaded_model.predict([{"product": "MLflow"}])
-        assert result == [TEST_CONTENT]
+        assert result == [{"text": TEST_CONTENT}]
```

Comment:
Can we change `"text"` to `model.output_key`? In case LangChain changes the output_key in the future, we won't need to modify out test. Besides, this explicitly says the "text" key is from the chain's output_key, instead of something returned by openai or something hard-coded by mlflow.
Wdyt?

Reply:
great suggestion to future proof this test! thanks